### PR TITLE
numbers

### PR DESCRIPTION
**Pull request by Volvo-hackangel**
what do the numbers mean mason

### CODE REVIEW

File: script/Ciel/Day 7/07_01_CIEL07_9.txt [hunk 1]
```diff
@@ -364,6 +364,7 @@ Rather, it was the complete opposite.
 [sha:612aba7de10bd7fafb85445437a8019d36c58d7b]{
 -- Page 675, Offset 16459.
 -- 彼は人をもっと知りたかった。千や二千では足りない。もっともっと多くの真実を集めたかった。
+// exception?
```
Comment (rschlaikjer) [inline script/Ciel/Day 7/07_01_CIEL07_9.txt:367]: yeah

File: script/Ciel/Day 9/09_01_CIEL09_2B.txt [hunk 2]
```diff
@@ -152,6 +152,7 @@ Is he also one of those top-tier vampires――Ancestors?"
 [sha:c378c60f25383d14b2c5d83c812e4a064275acf7]{
 -- Page 160, Offset 22536.
 -- 事実、教会は彼を二十八人目の祖として捉えています。他の祖たちは彼を同位と認めてはいませんが」
+// this one is a title right?
```
Comment (rschlaikjer) [inline script/Ciel/Day 9/09_01_CIEL09_2B.txt:155]: Hmm not sure, I think just the Twenty-Seven DAA is, but Petrikow can confirm

File: script/Ciel/Day 9/09_01_CIEL09_2B.txt [hunk 3]
```diff
@@ -887,6 +888,7 @@ It's clear from the cold look in her eyes what she's about to say.
 -- Page 211, Offset 22664.
 -- 彼女は八百年前に一度だけ過ちを犯した。真祖の王族でありながら人間の司祭だったロアの甘言に乗り、杯に汲まれたロアの血を飲んでしまった」
 // Wait, secondhand drinking counts?
+// exception?
```
Comment (rschlaikjer) [inline script/Ciel/Day 9/09_01_CIEL09_2B.txt:891]: Sure

File: script/Ciel/Day 9/09_01_CIEL09_4.txt [hunk 4]
```diff
@@ -994,6 +994,7 @@ How come nobody has ever wondered why she has a name like that...?
 [sha:95192f1cc176443303e309f93bb8a5e10021b8d5]{
 -- Page 440, Offset 23274.
 -- いくら秘蹟に耐性のない一般人でも、百人二百人に暗示をかけるなんて並大抵の魔力量じゃないもの。
+// exception?
```
Comment (rschlaikjer) [inline script/Ciel/Day 9/09_01_CIEL09_4.txt:997]: Sure

File: script/QA/QA_01.txt [hunk 5]
```diff
@@ -26,7 +26,7 @@ Welcome to 'Teach Me, Ciel-Sensei!'.%{n}
 [sha:cc86c2e84e7ec326d805dedf7a0261b93795a15b]{
 -- Page 4, Offset 42771.
 -- 教官のシエル（本編後、18歳想定）です。
-Instructor Ciel here.%{n}(Presumed to be 18 years old)
+Instructor Ciel here.%{n}(Presumed to be eighteen years old)
```
Comment (rschlaikjer) [inline script/QA/QA_01.txt:29]: You checked this fits?

File: script/QA/QA_22.txt [hunk 6]
```diff
@@ -136,7 +136,7 @@ buildings as well.
 [sha:83e4b627c7288fa8eb8247b26a8e1a30bb1e85a9]{
 -- Page 229, Offset 43739.
 -- 安定時になると全長は100メートル級で固定。
-When stabilized, the total length is fixed at 100 meters.
+When stabilized, the total length is fixed at hundred meters.
```
Comment (rschlaikjer) [inline script/QA/QA_22.txt:139]: one hundred, maybe keep as digits.

File: script/Ciel/Day 14/14_01B_03_CIEL14B_4_1.txt [hunk 7]
```diff
@@ -76,7 +76,7 @@ It's just as Arcueid says. The earth had turned into an environment that's unfit
 [sha:c2fe5c4aceb974e9ff76e6822d1c21072876b3c0]{
 -- Page 5, Offset 40260.
 -- 40度を超える高温と、今も成長し続け、少しずつ隙間を埋めていく緑の舌。
-The temperature passes over 40 degrees, and even now, the green tongue continues to grow and slowly cover the remaining gaps.
+The temperature passes over fourty degrees, and even now, the green tongue continues to grow and slowly cover the remaining gaps.
```
[the remark this answers is not in the document]
Comment (Volvo-hackangel) [reply script/Ciel/Day 14/14_01B_03_CIEL14B_4_1.txt:79]: I am dumb

File: script/Ciel/Day 2/02_01_CIEL02_1_2.txt [hunk 8]
```diff
@@ -488,11 +488,11 @@ It's not that I don't like them, it's more like I'm afraid to get too close to t
 [sha:55ed3923d2f8d0d3378de96ec32189098daf08ee]{
 -- Page 43, Offset 3727.
 -- きっかり１０００円分のポイントが貯まったカードを持て余す。
-The card with exactly 1,000 yen worth of points saved up on it weighs heavily upon me.
+The card with exactly a thousand yen worth of points saved up on it weighs heavily upon me.
```
[the remark this answers is not in the document]
Comment (Volvo-hackangel) [reply script/Ciel/Day 2/02_01_CIEL02_1_2.txt:491]: Going with the latter

File: script/Ciel/Day 9/09_01_CIEL09_2B.txt (hunk 2 above)
Comment (Volvo-hackangel) [reply script/Ciel/Day 9/09_01_CIEL09_2B.txt:155]: @Petrikow a little help?

File: script/QA/QA_01.txt (hunk 5 above)
Comment (Volvo-hackangel) [reply script/QA/QA_01.txt:29]: I will now

File: script/QA/QA_22.txt (hunk 6 above)
Comment (Volvo-hackangel) [reply script/QA/QA_22.txt:139]: what about a hundred?

File: script/QA/QA_01.txt (hunk 5 above)
Comment (Volvo-hackangel) [reply script/QA/QA_01.txt:29]: looks good to me
image

File: script/Arcueid/Day 9/09_00_ARC09_3_2_1.txt [hunk 9]
```diff
@@ -344,7 +344,7 @@ Was someone so unbearably uptight as Makihisa even capable of a thing like that?
 -- Page 173, Offset 21787.
 -- けど……うちの当主ってみんな短命なんだな。親父も五十前に病死した訳だし、その前は三十歳の時に事故死……その前は十八歳で自殺している……」
 But... all of the family heads have been surprisingly short-lived.
- My father passed away before he hit fifty, the one before him died in an accident at thirty-five...
+ My father passed away before he hit 50, the one before him died in an accident at 35...
  Their predecessor committed suicide at the age of eighteen..."
```
Comment (Petrikow) [inline script/Arcueid/Day 9/09_00_ARC09_3_2_1.txt:348]: eighteen can also be a number here just to match the other ones in this line

File: script/Ciel/Day 6/06_01_CIEL06_6.txt [hunk 10]
```diff
@@ -406,7 +406,7 @@ I never could have imagined that my treasured undying workshop would still funct
 [sha:8aa521c22ee330786eecc9f568c89c79986c23a6]{
 -- Page 250, Offset 13816.
 -- 『誇るべきですよ。貴方の魔道が八百年の技術革新に耐えるだけのものだった、という事ですから。
-'You should be proud. It means that your wizardry has been able to withstand eight hundred years of technical progress.
+'You should be proud. It means that your wizardry has been able to withstand 800 years of technical progress.
```
Comment (Petrikow) [inline script/Ciel/Day 6/06_01_CIEL06_6.txt:409]: technological progress

File: script/Ciel/Day 9/09_01_CIEL09_2B.txt (hunk 2 above)
Comment (Petrikow) [reply script/Ciel/Day 9/09_01_CIEL09_2B.txt:155]: 28th seems best here

**Pull request merged**